Modules/System/Cues and KPIs/src/CuesAndKPIs.Codeunit.al

```diff
@@ -93,4 +93,17 @@ codeunit 9701 "Cues And KPIs"
     internal procedure OnConvertStyleToStyleText(CueStyle: Enum "Cues And KPIs Style"; var Result: Text; var Resolved: Boolean)
     begin
     end;
+
+    /// <summary>
+    /// Event that allows definition of cue style for a cue using style enum without the usage of a cue setup table.
+    /// Subscribe to this event if you want to define a cue style for a cue using custom prerequisites.
+    /// </summary>
+    /// <param name="TableID">The ID of the table where the cue is defined.</param>
+    /// <param name="FieldNo">The ID of the field which the cue is based on.</param>
+    /// <param name="CueStyle">Exit parameter that holds newly determined cue style based on custom prerequisites.</param>
+    /// <param name="Resolved">A boolean value that describes whether or not the cue style has been determined.</param>
+    [IntegrationEvent(false, false)]
+    procedure OnBeforeGetCustomizedCueStyleOption(TableID: Integer; FieldNo: Integer; CueValue: Decimal; var CueStyle: Enum "Cues And KPIs Style"; var Resolved: Boolean)
```

Comment:
`CueValue` should be documented as well.

Reply:
Thanks, I missed the spot.